SCClassLibrary/Common/Streams/IOStream.sc

```diff
@@ -59,6 +59,9 @@ CollStream : IOStream {
 	pos_ { arg toPos;
 		pos = toPos.clip(0, collection.size);
 	}
+	rewind { |n = 1|
+		pos = max(0, pos - n);
+	}
```

Comment:
If you provide no argument, `stream.rewind`, sounds like "fully rewind". Couldn't the default be that?

```
rewind { |n|
if(n.isNil) { pos = 0 } {
		pos = max(0, pos - n)
}
}
```